src/IntervalSets.jl

```diff
@@ -283,6 +279,7 @@ include("findall.jl")
 
 if !isdefined(Base, :get_extension)
     include("../ext/IntervalSetsStatisticsExt.jl")
+    include("../ext/IntervalSetRandomExt.jl")
```

Comment:
```suggestion
    include("../ext/IntervalSetsRandomExt.jl")
    # We don't support the extension with StatsBase.jl before Julia v1.9.
    # include("../ext/IntervalSetsStatsBaseExt.jl")
```
* `IntervalSetRandomExt` should be `IntervalSetsRandomExt`.
* Adding the above comments will help other developers.